migrate/playbooks/rds-timescale-live-migration.md

```diff
@@ -372,32 +373,12 @@ Additionally, `follow` command exports a snapshot ID to `/tmp/pgcopydb/snapsho
 This ID can be utilized to migrate data that was in the database before the replication
 slot was created.
 
-## 4. Migrate roles and schema from source to target
-Before applying DML operations from the replication slot, the schema and data from
-the source database need to be migrated.
-The larger the size of the source database, the more time it takes to perform the
-initial migration, and the longer the buffered files need to be stored.
-
-### 4.a Migrate database roles from source database
-<LiveMigrationRoles />
-
-### 4.b Dump the database schema from the source database
-<DumpPreDataSourceSchema />
-
-### 4.c Load the roles and schema into the target database
-```sh
-psql -X -d "$TARGET" \
-  -v ON_ERROR_STOP=1 \
-  --echo-errors \
-  -f roles.sql \
-  -f pre-data-dump.sql
-```
+## 4. Perform Live Migration
 
-## 5. Perform "live migration"
 The remaining steps for migrating data from a RDS Postgres instance to Timescale
 with low-downtime are the same as the ones mentioned in "Live migration"
-documentation from [Step 5] onwards. You should follow the mentioned steps
+documentation from [Step 3] onwards. You should follow the mentioned steps
 to successfully complete the migration process.
 
 [live migration]: /migrate/:currentVersion:/live-migration/live-migration-from-postgres/
-[Step 5]: /migrate/:currentVersion:/live-migration/live-migration-from-postgres/#5-enable-hypertables
+[Step 3]: /migrate/:currentVersion:/live-migration/live-migration-from-postgres/#3-run-the-live-migration-docker-image
```

Comment:
This is not right. We are asking users to continue from Step 3 in `live-migration-from-postgres` page. The step 3 there is
```shell
docker run --rm -dit --name live-migration \
  -e PGCOPYDB_SOURCE_PGURI=$SOURCE \
  -e PGCOPYDB_TARGET_PGURI=$TARGET \
  -v ~/live-migration:/opt/timescale/ts_cdc \
  timescale/live-migration:v0.0.1
```
Since the above command already has `pgcopydb follow` internally, we should not ask users to perform step 3 in this page which also includes `pgcopydb follow`.

The correct way should be to ask users to continue from Step 3 in `live-migration-from-postgres` page right from step 3 on this page (by replacing step 3 on this page).